src/chainlit/server.py

```diff
@@ -46,6 +46,9 @@ async def lifespan(app: FastAPI):
             url = f"http://{host}:{port}"
 
         logger.info(f"Your app is available at {url}")
+
+        # Add a delay before opening the browser
```

Comment:
Is there a reason for the delay?

Reply:
Sometimes the server is not fully ready when the app opens in the browser. An error is then displayed for one second before the browser is able to make the connection. To avoid this we wait 1s